src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -472,9 +472,28 @@ def delete_lock(name, resource_group_name=None, resource_provider_namespace=None
 
 def create_lock(name, resource_group_name=None, resource_provider_namespace=None,
                 parent_resource_path=None, resource_type=None, resource_name=None,
-                level=None, notes=None, lock_id=None, lock_type=None):
+                level=None, notes=None, lock_id=None):
+    '''
+    :param name: The name of the lock.
+    :type name: str
+    :param resource_provider_namespace: Name of a resource provider.
+    :type resource_provider_namespace: str
+    :param parent_resource_path: Path to a parent resource
+    :type parent_resource_path: str
+    :param resource_type: The type for the resource with the lock.
+    :type resource_type: str
+    :param resource_name: Name of a resource that has a lock.
+    :type resource_name: str
+    :param level: The locking level, must be either 'ReadOnly' or 'CanNotDelete'
+    :type level: str
+    :param notes: Notes about this lock.
+    :type notes: str
+    '''
+    if level != 'ReadOnly' and level != 'CanNotDelete':
```

Comment:
Is there an SDK enum for these values instead of hardcoding them?
In _params.py the `level` param can be modified to take a list of possible values. Then you don't need this check and users will get tab completion.